glide.yaml

```diff
@@ -80,7 +80,7 @@ import:
   subpackages:
   - libcontainer/user
 - package: github.com/pkg/errors
-  version: a22138067af1c4942683050411a841ade67fe1eb
+  version: 839d9e913e063e28dfd0e6c7b7512793e0a48be9
```

Comment:
I assume this is to address conflicts with moby/buildkit?

Reply:
Without this, the test fails to compile with the error messages:
```
vendor/github.com/docker/docker/plugin/v2/plugin_linux.go:33:15: undefined: "github.com/openshift/source-to-image/vendor/github.com/pkg/errors".WithStack
vendor/github.com/docker/docker/plugin/v2/plugin_linux.go:105:16: undefined: "github.com/openshift/source-to-image/vendor/github.com/pkg/errors".WithStack
```